test/xmlcontent.js

```diff
@@ -1,64 +1,65 @@
-import parse from '../lib/index.js';
-import {expect} from 'chai';
+import parse from "../lib/index.js";
```

Comment:
Changes in this file are not relevant to this PR - exclude this file.

Reply:
this file contains change for conversion of var to let